pkg/ctl/cmdutils/configfile.go

```diff
@@ -440,8 +440,11 @@ func makeManagedNodegroup(nodeGroup *api.NodeGroup, options CreateManagedNGOptio
 }
 
 func validateManagedNGFlags(cmd *cobra.Command, managed bool) error {
-	flagsValidOnlyWithMNG := []string{"spot", "instance-types"}
+	if managed {
+		return nil
+	}
```

Comment:
why is the func call `validateManagedFlags` and a bool is passed in to indicate its managed? Is it not called because its validating flags for managed nodes? :see_no_evil:

Reply:
i think it was named to validate the use of managed flags (not managed _nodegroup_ flags), now it only checks for the use of managed flags on unmanaged ngs

it is confusing because technically it is named accurately, but we use "managed" as a shorthand for "managed nodegroups" so our minds fill in a gap which is not there

Reply:
ohhhh. Gotcha. Since for managed its always `if managed; return nil` would it make sense to move that logic outside of the func? Might make it less confusing

> i think it was named to validate the use of managed flags (not managed _nodegroup_ flags), now it only checks for the use of managed flags on unmanaged ngs

it does have NG in the func name though :wink:

Reply:
> would it make sense to move that logic outside of the func? Might make it less confusing

i did think about that, and decided to do the `if thing` in one place rather than more than 1, don't really care tho 🤷‍♀️ 

> it does have NG in the func name though

oh shit yeh, what was i reading 🤣